apps/docs-app/app/components/f/components/tooltip.gts

```diff
@@ -0,0 +1,206 @@
+import { array, fn } from '@ember/helper';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { Toaster, Tooltip } from '@nrg-ui/core';
+import FreestyleUsage from 'ember-freestyle/components/freestyle/usage';
+import FreestyleSection from 'ember-freestyle/components/freestyle-section';
+
+import CodeBlock from '../../code-block';
+
+import type { Alignment, Side } from '@floating-ui/dom';
+import type { ToastService } from '@nrg-ui/core/services/toast';
+
+export default class extends Component {
+  @service
+  declare toast: ToastService;
+
+  @tracked
+  alignment?: Alignment;
+
+  @tracked
+  arrow: boolean = true;
+
+  @tracked
+  isShown?: boolean;
+
+  @tracked
+  offset?: number;
+
+  @tracked
+  side?: Side;
+
+  update = (key: string, value: unknown) => {
+    this[key] = value;
+  };
+
+  log = (...args: unknown[]) => {
+    console.log(...args);
+  };
+
+  <template>
+    <Toaster />
+    <FreestyleSection @name="Tooltip" as |Section|>
+      <Section.subsection @name="Basics">
+        <FreestyleUsage>
+          <:example>
+            <div class="p-2">
+              <Tooltip
+                @alignment={{this.alignment}}
+                @arrow={{this.arrow}}
```

Comment:
`arrow` is specified as an argument for the tooltip here, but isn't wired up in the actual component.